build/linux/installer/conf/fluent-bit-input.conf

```diff
@@ -7,3 +7,12 @@
     Name perf
     Tag  oneagent.containerInsights.LINUX_PERF_BLOB
     run_interval 60
+
+[INPUT]
+    Name        tcp
+    Tag         mdm.container.perf.telegraf.*
+    Listen      0.0.0.0
+    Port        25228
```

Comment:
I didnt see any code either to disable fluentd config related to tcp inout with 25528 or condition to activate new code.where are we doing this code should be activated enabled on resource optimization enabled flag?

Reply:
the code becomes active when either resource_optimization is enabled from CM or through toggle. Let me add the CM part back.

When the correct variable is passed fluentd gets disabled